test/dotnet-add-p2p.Tests/GivenDotnetAddP2P.cs

```diff
@@ -477,7 +493,7 @@ public void WhenProjectNameIsNotPassedItFindsItAndAddsReference()
         [Fact]
         public void ItAddsRefBetweenImports()
         {
-            var lib = NewLib();
+            var lib = NewLibWithFrameworks();
```

Comment:
Not in this PR, but I want to take a pass through these new abstractions. I'm seeing a new instance of a pattern we're just now starting to get rid of... things that should be composed with e.g. Fluent extension methods are instead turning into a combanitronics problem. This eventually becomes NewLibWithFrameworksNoRidFourReferences()... Don't fix it in this PR but we'll need to figure out how to clean up soon.